endpointmanager/pkg/capabilityparser/factory.go

```diff
@@ -78,5 +78,5 @@ func NewCapabilityStatementFromInterface(capStat map[string]interface{}) (Capabi
 		return newR4(capStat), nil
 	}
 
-	return nil, fmt.Errorf("unknown FHIR version %s", fhirVersion)
+	return newDSTU2(capStat), fmt.Errorf("unknown FHIR version, %s, defaulting to DSTU2", fhirVersion)
```

Comment:
I think we should do the log.Warn _here_  indicating that  there was an unknown fhirVersion so we are  defaulting to using the DSTU2 version and return the newDSTU2(capStat) without error.